AsyncDisplayKit/ASMapNode.mm

```diff
@@ -265,17 +277,23 @@ - (void)addLiveMap
     [_mapView addAnnotations:_annotations];
     [weakSelf setNeedsLayout];
     [weakSelf.view addSubview:_mapView];
-    
-    if (CLLocationCoordinate2DIsValid(_centerCoordinateOfMap)) {
-      [_mapView setCenterCoordinate:_centerCoordinateOfMap];
+
+    if (self.showAnnotationsOptions & ASMapNodeShowAnnotationsOptionsZoomed) {
+      BOOL const animated = self.showAnnotationsOptions & ASMapNodeShowAnnotationsOptionsAnimated;
+      [_mapView showAnnotations:_mapView.annotations animated:animated];
     }
+    
+//    if (CLLocationCoordinate2DIsValid(_centerCoordinateOfMap)) {
```

Comment:
That particular line is not needed because of line 267 `[weakSelf applySnapshotOptions];` in that method the region is set.

There is no reason to keep it commented out, I will delete it.